tools/DARIAH-DE-Geo-Browser-CSV.json

```diff
@@ -0,0 +1,34 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
+    "name": "DARIAH-DE Geo-Browser (CSV)",
+    "logo": "dariah-de.svg",
+    "homepage": "https://de.dariah.eu/en/geobrowser",
+    "location": "Göttingen, Germany",
+    "creators": "DARIAH-DE",
+    "contact": {
+        "person": "DARIAH-DE Helpdesk",
+        "email": "support@de.dariah.eu"
+    },
+    "version": "v3.6-BETA",
+    "authentication": "no",
+    "description": "The DARIAH-DE Geo-Browser allows a comparative visualization of several requests and facilitates the representation of data and their visualization in a correlation of geographic spatial relations at corresponding points of time and sequences. Thus, researchers can analyze space-time relations of data and collections of source material and simultaneously establish correlations between them.",
+    "licence": "LGPL-3.0-or-later",
+    "output": [
+        "none"
+    ],
+    "mimetypes": [
+        "text/csv",
+        "text/comma-separated-value",
+        "application/vnd.dariahde.geobrowser.csv"
+    ],
+    "languages": [],
+    "langEncoding": "639-1",
+    "url": "https://geobrowser.de.dariah.eu/beta/",
```

Comment:
Are we going to link to a beta to our production instance? Do we have an idea about the maturity and stability of the beta?